apps/telegram-bot/core/services/file_id_capture.py

```
@@ -0,0 +1,78 @@
+    if not file_id_updates:
+        return
+
+    try:
+        r = await _get_redis()
+        payload = json.dumps({
+            "metadata_url": metadata_url,
+            "file_id_updates": file_id_updates,
+        }, ensure_ascii=False)
+        await r.lpush(FILEID_QUEUE_KEY, payload)
+        logger.info(f"Pushed {len(file_id_updates)} file_id updates for {metadata_url}")
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)